src/components/Head.css

```diff
@@ -27,11 +27,6 @@
   height: 66px;
 }
 
-/* style the wrapper div around the map that we don't seem to be able to assign a class name to  */
-.mobile-menu + div {
-  height: 100%;
-}
-
```

Comment:
Don't need some of this mobile specific stuff anymore. Instead I added a new wrapper div around the parent map object.